library/src/main/java/com/vijay/jsonwizard/presenters/JsonFormFragmentPresenter.java

```diff
@@ -275,24 +270,7 @@ public ValidationStatus writeValuesAndValidate(LinearLayout mainView) {
         for (int i = 0; i < childCount; i++) {
             View childAt = mainView.getChildAt(i);
             String key = (String) childAt.getTag(R.id.key);
-            if (childAt instanceof MaterialEditText) {
```

Comment:
This is not required anymore?

Reply:
Yes, it's not required. Edit texts use the MaterialTextInputLayout obj, instead of the MaterialEditText one.